### PR TITLE
Feature/AUT-120/support parsing figcaption element

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/AUT-120
Depends on: 

- [x] https://github.com/oat-sa/extension-tao-itemqti/pull/2138
- [x] https://github.com/oat-sa/tao-core/pull/3540

For local test need to use brach from qti-sdk, please run  `composer require "qtism/qtism:dev-feature/legacy/TR-1499/support_qti_figure_figcaption_elements as 0.25.2" -W`

# How to test:

- create new passage
- add image to it
- add/edit image caption
- save
- check that it is rendered properly in preview, in items

### CODE REVIEW

File: views/js/qtiCreator/widgets/static/figure/states/states.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2022 (original work) Open Assessment Technologies SA ;
+ *
+ */
+define([
+    'taoQtiItem/qtiCreator/widgets/states/factory',
+    'taoQtiItem/qtiCreator/widgets/static/states/states',
+    'taoQtiItem/qtiCreator/widgets/static/figure/states/Sleep',
+    'taoMediaManager/qtiCreator/widgets/static/figure/states/Active'
```
Comment (marpesia) [inline views/js/qtiCreator/widgets/static/figure/states/states.js:23]: Does the file on `taoQtiItem/views/js/qtiCreator/widgets/static/figure/states/Active.js` the same as on `'taoMediaManager/qtiCreator/widgets/static/figure/states/Active'` ?

**Review by marpesia — CHANGES_REQUESTED** [review]
The FE works great.

I've checked locally, and I found some issues:
- Add some text paragraphs
- Add an Image at the end of a paragraph line
- Uncheck the Responsive mode of Img
The Height is NaN, but it is working. If I change the Width, it changes the Hight.
I've checked if the Img is added out of the paragraph zone and it doesn't happen. If I first add the Image at the end, and later a second in the paragraphs, it doesn't happen too.
image

I can't save the Asset if it contains an Img, I don't know if I missed something on the installation.
image

In Kitchen, on adding an Image. Can you see the same?
image

File: views/js/qtiCreator/widgets/static/figure/states/states.js (hunk 1 above)
Comment (marpesia) [reply views/js/qtiCreator/widgets/static/figure/states/states.js:23]: Update: I can't see the NaN issue on Items on Kitchen, but I see it on Items on Local. So maybe there is something wrong with my local with that.

Comment (marpesia) [reply views/js/qtiCreator/widgets/static/figure/states/states.js:23]: Update, adding this extension I can reproduce local and kitchen the same issue on Items and Assets.
It is the 1st time you unchecked the 'Responsive mode', the functionality is there, it is just the 1st time, and there is no correct value for that field.
image

**Comment by Silvia-loza** [comment]
Code looks good and most things work as expected in local , however found this edge case in passage. :arrow_down: :arrow_down: 
image

**Review by marpesia — APPROVED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [ ] Pull request's target is not `master`
- [ ] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Commits messages are meaningful
- [ ] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by Silvia-loza — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X]  Pull request's target is not` master`
- [X] New code is not subject to concurrency issues (if applicable)
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful
